app/admin/logs/api_logs.rb

```diff
@@ -37,6 +47,8 @@ def find_resource
   filter :response_body
   filter :request_headers
   filter :response_headers
+  filter :remote_ip_eq_inet, as: :string, label: 'Remote IP'
+  filter :current_customer_id_equal, as: :string, label: 'Current Customer ID Equal'
```

Comment:
no need to add filtering by meta, it can be anything info, not necessarily the customer. filtering by remote_ip will be enough